lighthouse-plugin-publisher-ads/audits/ad-blocking-tasks.js

```diff
@@ -8,25 +8,35 @@
 // limitations under the License.
 
 const AdRequestTime = require('../computed/ad-request-time');
+const common = require('../messages/common-strings');
 const LongTasks = require('../computed/long-tasks');
 const {auditNotApplicable} = require('../utils/builder');
-const {AUDITS, NOT_APPLICABLE} = require('../messages/messages');
 const {Audit} = require('lighthouse');
-const {formatMessage} = require('../messages/format');
 const {getAttributableUrl} = require('../utils/tasks');
 const {isGpt} = require('../utils/resource-classification');
 const {URL} = require('url');
-
-const id = 'ad-blocking-tasks';
-const {
-  title,
-  failureTitle,
-  description,
-  displayValue,
-  failureDisplayValue,
-  headings,
-} = AUDITS[id];
-
+// @ts-ignore
+const i18n = require('lighthouse/lighthouse-core/lib/i18n/i18n.js');
+
+const UIStrings = {
+  /* Title of the audit */
+  title: 'No long tasks blocking ad-related network requests',
+  failureTitle: 'Avoid long tasks that block ad-related network requests',
+  description: 'Tasks blocking the main thread can delay ad requests and cause ' +
+  'a poor user experience. Consider removing long blocking tasks or moving ' +
+  'them off of the main thread. These tasks can be especially detrimental to ' +
+  'performance on less powerful devices. [Learn more](' +
+  'https://developers.google.com/publisher-ads-audits/reference/audits/ad-blocking-tasks' +
+  ').',
+  failureDisplayValue: '{numTasks} long {numTasks, plural, =1 {task} other ' +
+  '{tasks}}',
+  columnScript: 'Attributable URL',
+  columnStartTime: 'Start',
+  columnDuration: 'Duration',
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename,
```

Comment:
To avoid duplication of importing `common`, maybe there should be some wrapper helper method for us?